### PR TITLE
fix paging

### PR DESCRIPTION
**Pull request by twoinke**

### CODE REVIEW

File: steckos/shell/shell.asm [hunk 1]
```diff
@@ -985,7 +986,7 @@ dir_show_entry_long:
 @y:
 
         jsr print_fat_time
-        crlf
+        ; crlf
```
Comment (mlauke) [inline steckos/shell/shell.asm:989]: a comment?

File: steckos/shell/shell.asm [hunk 2]
```diff
@@ -1167,32 +1169,32 @@ dir:
...
         cmp #13 ; enter pages line by line
         beq @lx
 
+        cmp #KEY_ESCAPE
+        beq @end
         ; check ctrl c
-        bit flags
-        bmi @exit
+        ; bit flags
+        ; bmi @end
```
Comment (mlauke) [inline steckos/shell/shell.asm:1190]: another comment ? :yum:

File: steckos/shell/shell.asm [hunk 3]
```diff
@@ -1405,7 +1407,7 @@ pd_header:      .asciiz "####   0  1  2  3  4  5  6  7  8  9  A  B  C  D  E  F
 pattern:        .byte "*.*",$00
 attr_tbl:       .byte DIR_Attr_Mask_ReadOnly, DIR_Attr_Mask_Hidden,DIR_Attr_Mask_System,DIR_Attr_Mask_Archive
 attr_lbl:       .byte 'R','H','S','A'
-press_key_msg:  .byte "-- press a key-- ",$00
+press_key_msg:  .byte $0a, $0d,"-- press a key-- ",$00
```
Comment (mlauke) [inline steckos/shell/shell.asm:1410]: $0a, $0d => maybe CODE_LF is sufficient

File: steckos/shell/shell.asm (hunk 2 above)
Comment (twoinke) [reply steckos/shell/shell.asm:1190]: yes

File: steckos/shell/shell.asm (hunk 3 above)
Comment (twoinke) [reply steckos/shell/shell.asm:1410]: no

File: steckos/shell/shell.asm (hunk 1 above)
Comment (mlauke) [reply steckos/shell/shell.asm:989]: remove commented code.

File: steckos/shell/shell.asm (hunk 2 above)
Comment (twoinke) [reply steckos/shell/shell.asm:1189]: no

**Review by mlauke — APPROVED** [review]

**Pull request merged**